tests/src/resultcollection.hh

```diff
@@ -0,0 +1,170 @@
+                                .addResultRequest(ResultType::linearStress);
+
+  return std::make_tuple(resultRequirements, expectedStress, getVertexPositions(fe));
+};
+
+inline auto linearStressResultsOfCube = [](auto& nonLinearOperator, auto& fe) {
+  constexpr int nodes      = 8;
+  constexpr int quantities = 6;
+
+  Eigen::Matrix<double, nodes, quantities> expectedStress;
+  expectedStress << 576.92307692, 1346.15384615, 576.92307692, 384.61538462, 0, 384.61538462, 0, 0, 0, 0, 0, 0,
+      -1346.15384615, 192.30769231, -1346.15384615, 0, -769.23076923, 0, -1346.15384615, -576.92307692, -576.92307692,
+      0, -384.61538462, -384.61538462, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -576.92307692, -576.92307692, -1346.15384615,
+      -384.61538462, -384.61538462, 0, 0, 0, 0, 0, 0, 0;
+
+  auto& displacement = nonLinearOperator.firstParameter();
+  displacement << 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0;
+
+  using namespace Ikarus;
+  auto resultRequirements = ResultRequirements()
+                                .insertGlobalSolution(FESolutions::displacement, displacement)
+                                .addResultRequest(ResultType::linearStress);
+
+  return std::make_tuple(resultRequirements, expectedStress, getVertexPositions(fe));
+};
+
+inline auto linearStressResultsOfTriangle = [](auto& nonLinearOperator, auto& fe) {
+  constexpr int nodes      = 3;
+  constexpr int quantities = 3;
+
+  Eigen::Matrix<double, nodes, quantities> expectedStress;
+  expectedStress.rowwise() = Eigen::Matrix<double, 1, quantities>{2197.80219780, 659.34065934, 384.61538462};
+
+  auto& displacement = nonLinearOperator.firstParameter();
+  displacement << 0, 0, 2, 0, 1, 0;
+
+  using namespace Ikarus;
+  auto resultRequirements = ResultRequirements()
+                                .insertGlobalSolution(FESolutions::displacement, displacement)
+                                .addResultRequest(ResultType::linearStress);
+
+  return std::make_tuple(resultRequirements, expectedStress, getVertexPositions(fe));
+};
+
+inline auto linearStressResultsOfTetrahedron = [](auto& nonLinearOperator, auto& fe) {
+  constexpr int nodes      = 4;
+  constexpr int quantities = 6;
+
+  Eigen::Matrix<double, nodes, quantities> expectedStress;
+  expectedStress.rowwise()
+      = Eigen::Matrix<double, 1, quantities>{576.92307692, 1346.15384615, 576.92307692, 0, 384.61538462, 769.23076923};
+
+  auto& displacement = nonLinearOperator.firstParameter();
+  displacement << 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0;
+
+  using namespace Ikarus;
+  auto resultRequirements = ResultRequirements()
+                                .insertGlobalSolution(FESolutions::displacement, displacement)
+                                .addResultRequest(ResultType::linearStress);
+
+  return std::make_tuple(resultRequirements, expectedStress, getVertexPositions(fe));
+};
+
+//
+// auto createLinearStressResults = []<typename NonLinearOperator, typename FiniteElement>(
```

Comment:
I would remove all the commented code here before merging

Reply:
Already did that

Reply:
Ah ok. Thanks.